src/cleanup.ts

```diff
@@ -18,9 +19,9 @@ async function cleanup() {
     }
 }
 
-function addCachedCliToPath(): boolean {
+function addCachedJfToPath(): boolean {
     // Get the JFrog CLI path from step state. saveState/getState are methods to pass data between a step, and it's cleanup function.
-    const jfrogCliPath: string = core.getState(Utils.JFROG_CLI_PATH_STATE);
+    const jfrogCliPath: string = core.getState('jf');
```

Comment:
It seems you're saving the state using the actual file name:
```ts
core.saveState(fileName, cliCacheDir);
```
But you're reading the state using the 'jf' key:
```ts
core.getState('jf');
```

This approach won't work on Windows. If you agree, let's stick to the `Utils.JFROG_CLI_PATH_STATE` key convention.